### PR TITLE
Stats: Follow up on the feedback modal settings

### PR DESCRIPTION
**Pull request by dognose24**
Related to https://github.com/Automattic/wp-calypso/pull/94197

## Proposed Changes

* Refactor the notice ID `able_to_submit_user_feedback`.
* Fix the dismiss link layout on Odyssey Stats.
* Update the feedback submission throttle to 5 minutes.

## Why are these changes being made?

* As a follow-up PR for https://github.com/Automattic/wp-calypso/pull/94197.

## Testing Instructions

* Spin this change along with testing instructions in https://github.com/Automattic/jetpack/pull/39287.
* Ensure the `Dismiss` link on the floating feedback panel aligns with the one on Calypso Stats.

|Before|After|
|-|-|
|截圖 2024-09-11 上午2 18 54|截圖 2024-09-11 上午2 19 41|

* Ensure each feedback form submission limit is set to 5 minutes.

截圖 2024-09-11 上午2 40 03

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-aUh-p2)?

### CODE REVIEW

File: client/my-sites/stats/hooks/use-notice-visibility-query.ts [hunk 1]
```diff
@@ -14,7 +15,7 @@ const DEFAULT_SERVER_NOTICES_VISIBILITY = {
 	// TODO: Check if the site needs to be upgraded to a higher tier on the back end.
 	tier_upgrade: true,
 	gdpr_cookie_consent: false,
-	able_to_submit_user_feedback: true,
+	[ NOTICE_KEY_FOR_FEEDBACK_SUBMISSION ]: true,
 	[ NOTICES_KEY_SHOW_FLOATING_USER_FEEDBACK_PANEL ]: true,
```
Comment (a8ck3n) [inline client/my-sites/stats/hooks/use-notice-visibility-query.ts:19]: We should probably keep the const naming consistent. I used `NOTICES_KEY` and you've used `NOTICE_KEY`. I'd also like the keys to be a bit more descriptive. If possible, maybe `NOTICES_KEY_ABLE_TO_SUBMIT_FEEDBACK`?

File: client/my-sites/stats/feedback/style.scss [hunk 2]
```diff
@@ -121,7 +121,7 @@
 	font-weight: 500;
 	line-height: 21px;
 
-	&.components-button {
+	&.components-button.is-link {
```
Comment (a8ck3n) [inline client/my-sites/stats/feedback/style.scss:124]: This fixes the link margin in my Odyssey testing.

**Review by a8ck3n — APPROVED** [review]
This tests well for me. I'm able to:

1. See the correct margin on the dismiss link button.
2. Submit feedback from Odyssey.
3. …and see the 5 min message after submission.
4. Waiting 5 mins and opening the modal again allows for another submission.

**Comment by a8ck3n** [comment]
One other note. The width of the modal is almost full screen on my laptop which, I believe, differs from the width when testing in Calypso. We should probably follow up with a max width to keep the text line-length to something reasonable. At the current width, it's a bit difficult to read.

**Comment by dognose24** [comment]
> The width of the modal is almost full screen on my laptop

That is the issue of the assets import sequence on Odyssey Stats. Addressed in [ef1f101](https://github.com/Automattic/wp-calypso/pull/94382/commits/ef1f1015cdadbc1ccf25d2321fe89e85aa68a5c8).

File: client/my-sites/stats/hooks/use-notice-visibility-query.ts (hunk 1 above)
Comment (dognose24) [reply client/my-sites/stats/hooks/use-notice-visibility-query.ts:19]: Updated in [ef1f101](https://github.com/Automattic/wp-calypso/pull/94382/commits/ef1f1015cdadbc1ccf25d2321fe89e85aa68a5c8).

**Review by kangzj — APPROVED** [review]
LGTM

**Comment by a8ck3n** [comment]
Looks good to me!

**Pull request merged**